UI/Web/src/app/nav/_components/grouped-typeahead/grouped-typeahead.component.scss

```diff
@@ -17,7 +17,7 @@ form {
 
 .typeahead-input {
     border: 1px solid transparent;
-    border-radius: 4px;
+    border-radius: 50px;
```

Comment:
Let's revert this back to 5px as pill shape does not fit within the UX.

Reply:
Reverting back to 4px, thinking about adding a css variable so it can be changed in a theme for users if wanted